packages/xrpl/src/models/transactions/AMMBid.ts

```diff
@@ -0,0 +1,85 @@
+import { ValidationError } from '../../errors'
+import { Amount } from '../common'
+
+import { BaseTransaction, isAmount, validateBaseTransaction } from './common'
+
+const MAX_AUTH_ACCOUNTS = 4
+
+interface AuthAccount {
+  AuthAccount: {
+    Account: string
+  }
+}
+
+/**
+ * AMMBid is used for submitting a vote for the trading fee of an AMM Instance.
+ *
+ * Any XRPL account that holds LPToken for an AMM instance may submit this
+ * transaction to vote for the trading fee for that instance.
+ */
```

Comment:
This description is the one for `AMMVote`, not `AMMBid` - AMMBid is for bidding on the continuous auction mechanism for the 0-fee arbitrage slot.